scripts/entrypoint.sh

```diff
@@ -36,7 +44,8 @@ mkdir -p /openair-hss/logs
 
 # Create the certificates
 pushd /openair-hss/scripts
-./data_provisioning_users --apn ${APN1} --apn2 ${APN2} --key ${LTE_K} --imsi-first ${FIRST_IMSI} --msisdn-first 00000001 --mme-identity oai-mme.${REALM} --no-of-users ${NB_USERS} --realm ${REALM} --truncate False --verbose True --cassandra-cluster ${cassandra_Server_IP}
+./wait-for-it.sh  ${cassandra_Server_IP}:9042
```

Comment:
@lionelgo 

can you please add a comment on the reason for port `9042`?

Reply:
It is the serving port for Apache Cassandra native protocol clients